products/compute/api.yaml

```diff
@@ -9452,11 +9452,11 @@ objects:
                           - :TEMPORARY_REDIRECT
                       - !ruby/object:Api::Type::Boolean
                         name: 'stripQuery'
-                        default_value: false
+                        required: true
```

Comment:
There should be 8 occurrences of `stripQuery` in total:
* `pathMatchers.routeRules.urlRedirect.stripQuery`
* `pathMatchers.pathRules.urlRedirect.stripQuery`
* `pathMatchers.pathRules.defaultUrlRedirect.stripQuery`
* `defaultUrlRedirect.stripQuery`

And these for both for `RegionUrlMap` and `UrlMap`.

Reply:
There are, but two of them were not marked as `required` before. Marking them as required would be a breaking change that can't happen on a minor version

Reply:
OK. But then it should be noted somewhere, that these two blocks where the non-required `stripQuery` properties appear exhibit the "empty block" problem that @danawillow pointed out.